tests/test_requester.py

```diff
@@ -0,0 +1,16 @@
+from unittest import TestCase
+
+from nextcloud.requester import Requester, NextCloudConnectionError
+
+
+class TestRequester(TestCase):
+
+    def test_wrong_url(self):
+        req = Requester('http://wrong-url.wrong', 'user', 'password', json_output=True)
+        exception_raised = False
+        try:
+            req.get('')
+        except NextCloudConnectionError as e:
+            exception_raised = True
```

Comment:
Please add a check here that the `wrong-url` is part of the exception message.

Reply:
Done

Reply:
Sorry for making it longer, but I would go for a more robust approach. According to what I think (correct me if you think that it is not the case):

* The recommended way how to turn an exception to the message is to convert the whole exception instance to string.
* The error message may be composed slightly differently in the future, and we don't want tests to fail because of this. I would therefore go for `wrong_url in str(e)`

Reply:
You right, it will be more robust for future changes. Updated it.